cmd/xl-v1-utils.go

```diff
@@ -42,14 +41,20 @@ func reduceErrs(errs []error, ignoredErrs []error) error {
 		errorCounts[err]++
 	}
 	max := 0
-	var errMax error
 	for err, count := range errorCounts {
 		if max < count {
 			max = count
-			errMax = err
+			maxErr = err
+		}
+	}
+	if maxErr != nil {
```

Comment:
Even if ``maxErr`` is ``nil`` the count must be greater than or equal to the quorum value.